### ISSUE TITLE
Noise generation screws up on pretty big seeds. Output is randomly raised or lowered by large amounts suddenly and small artifacts appear if that doesn't happen. Usually happens when seed is >500.

### ISSUE THREAD

**Issue by earth418** [devdad/SimplexNoise#12]

**Comment by devdad** [comment]
Hi thanks man for info. Can you please make some screenshot and give me some more details for example which functions are you using and did you pull latest version?
Thanks in advance!

**Comment by earth418** [comment]
Sure! I'm using C++, SimplexNoise2D and I'm using an int32 as the seed, which is added to a location (FVector2D) and some other math is involved as well. I can send a bit of code later, as I'm not home. I've attached a few pictures of varying seeds. HighresScreenshot00005 is the normal land, with a relatively low seed (about 100). Unknown-28 is at about 500, unknown-30 is at about 1000 afaik, and unknown-27 is 10000. The numbers are different for different sized lands. These were for lands 1024x1024 in resolution, and I've tested the same seeds on smaller sizes lands ie 512x512 and 256x256, and the artifacts appear much later on smaller sized meshes. I'm not sure if the error is in the SimplexNoise2D function, but i don't know what else it could be (and why it's related to the size and seed of the generated land). 

I don't think I've pulled the most recent version. I'll try that.

Another issue I have is that every time I compile it makes a new .dll file and doesn't delete the old one. They build up, and at one point they took up nearly 25 gigabytes. I'll see if the new version of the plugin fixes that.

[
![unknown-28](https://user-images.githubusercontent.com/42591855/64057532-2a69e400-cb6c-11e9-9eb4-c8f8e8f42d22.png)
![unknown-30](https://user-images.githubusercontent.com/42591855/64057536-335ab580-cb6c-11e9-982a-1e17486b4a4d.png)
![unknown-27](https://user-images.githubusercontent.com/42591855/64057537-381f6980-cb6c-11e9-9309-a644893ce444.png)
![HighresScreenshot00005](https://user-images.githubusercontent.com/42591855/64057582-a82def80-cb6c-11e9-80be-759d0397ee82.png)



](url)
